operator/option/config.go

```diff
@@ -250,6 +252,10 @@ const (
 	// SetCiliumIsUpCondition sets the CiliumIsUp node condition in Kubernetes
 	// nodes.
 	SetCiliumIsUpCondition = "set-cilium-is-up-condition"
+
+	// AllowedAnnotations are the annotations which are needed to propogate
+	// from Ingress to the Load Balancer
+	AllowedAnnotations = "allowed-annotations"
```

Comment:
from the end-user perspective, I think allowed-annotations name is a little bit confusing due to:
- a little bit generic, not all users will check the description.
- the allowed word seems to be misleading. All annotations are allowed, but only some annotations are propagated to Service LB.

I don't know what is the best name here, what do think about ingress-lb-annotations?